src/server.js

```diff
@@ -29,6 +31,29 @@ export const startServer = port => new Promise((resolve, reject) => {
 
   app.use('/api/v1', indexRouter);
 
+  app.post('/api/v1/articles/:articleId/ratings', async (req, res) => {
+    try {
```

Comment:
The callback should be in the controller. The route should be defined in the router file

Reply:
Yea, I was only testing it out. I was having some bug when I ran my test against the models. I will refactor the code. Thanks for pointing it out